content/api/entity-reference/fundraising/index.html

```diff
@@ -68,7 +68,7 @@
             <tr>
                 <td>inactive</td>
                 <td>boolean</td>
-                <td>Indicates whether the appeal is active.</td>
+                <td>Indicates whether the appeal is active. This is a computed field based on <code>start_date</code> and <code>end_date</code> properties.</td>
```

Comment:
I'm not a fan of the "based on" pattern we use here. It suggests how the field works but isn't explicit. I think that if we are going to call the field out as a computed field, we should explain how it computes the value. In this case, I assume that if there is a start_date but no end_date, then it is active. Otherwise (i.e., if there is an end_date or there is no start_date), then it is inactive. Is that right? If so, I'd change this to "This computed field indicates that the appeal is active if it has a <code>start_date</code> but no <code>end_date</code> Otherwise, it indicates that the appeal is inactive."